packages/foundry/Makefile

```
@@ -0,0 +1,84 @@
+.PHONY: build deploy generate-abis verify-keystore account chain compile deploy-verify flatten fork format lint test verify
+
+
+# Start local chain
+chain:
+	shx rm ~/.foundry/keystores/scaffold-eth-default 2>/dev/null; \
+	cast wallet import --private-key 0x2a871d0798f97d79848a013d4936a73bf4cc922c825d33c1cf7073dff6d409c6 --unsafe-password 'localhost' scaffold-eth-default && \
+	anvil
+
+# Build the project
+build:
+	forge build --build-info --build-info-path out/build-info/
+
+# Deploy the project
+deploy:
+	@if [ "$(RPC_URL)" = "localhost" ]; then \
+		forge script script/Deploy.s.sol --rpc-url localhost --password localhost --broadcast --legacy --ffi; \
+	else \
+		forge script script/Deploy.s.sol --rpc-url $(RPC_URL) --broadcast --legacy --ffi; \
+	fi
+
+# Build and deploy target
+build-and-deploy: build deploy generate-abis
+
+# Generate TypeScript ABIs
+generate-abis:
+	node scripts-js/generateTsAbis.js
+
+# Verify keystore
+verify-keystore:
+	@cast wallet address 
+
+# List account
+account:
+	@node scripts-js/ListAccount.js $$(make verify-keystore)
+
+# Generate a new account
+account-generate:
+	@cast wallet import $${1:-scaffold-eth-custom} --private-key $$(cast wallet new | grep 'Private key:' | awk '{print $$3}')
+	@echo "Please update .env file with ETH_KEYSTORE_ACCOUNT=$${1:-scaffold-eth-custom}"
+
```

I think missed the ACCOUNT_NAME from `package.json`
```suggestion
	@cast wallet import $(ACCOUNT_NAME) --private-key $$(cast wallet new | grep 'Private key:' | awk '{print $$3}')
	@echo "Please update .env file with ETH_KEYSTORE_ACCOUNT=$(ACCOUNT_NAME)"
```